docs/onboarding.md

```diff
@@ -0,0 +1,57 @@
+# Welcome to OpenTransit!
+
+## Project summary
+
+OpenTransit is dedicated to empowering people around the world to work with local governments to improve public transit using data. The project originated at the [San Francisco Brigade](https://muni.opentransit.city/) and we extended it to [PDX](http://opentransit-youngj.herokuapp.com/). We’re doing this by visualizing real-time data of the TriMet system so riders understand bus speed and reliability, and using the data to improve the TriMet system.
+
+## Get started
+
+1. Join [Code for PDX Slack](http://www.codeforpdx.org/welcome).
+2. Post your email and GitHub name on the **#opentransit-pdx** channel and we’ll add you to our [GitHub](https://github.com/codeforpdx/opentransit-metrics) and [Google Drive](https://drive.google.com/drive/folders/10usf0ddMH1Z8Q06ZsHqacKROm3xZ4NZe?usp=sharing).
+3. Try our [PDX MVP demo](http://opentransit-youngj.herokuapp.com/). What are the stats on your most common bus trip?
+
+## Ready to contribute?
+
+1. Get [opentransit-metrics](https://github.com/codeforpdx/opentransit-metrics) cloned and running on your computer.
+2. Play around with [opentransit-metrics](https://github.com/codeforpdx/opentransit-metrics) and see if you find any problems with the data or functionality. [File an issue](https://github.com/codeforpdx/opentransit-metrics/issues) with the problem you found.
+3. [Find a GitHub issue labeled “Good First Issue,”](https://github.com/codeforpdx/opentransit-metrics/labels/Good%20First%20Issue) ideally one without anyone already assigned to it. Take a crack at it — ask on Slack if you have a question!
+
+- If someone’s already assigned to a task you want to work on, ask the team if that person is still working on it — sometimes people move on to other tasks, so you may be able to pick it up anyway.
+
+4. Join our virtual meeting on Thursdays at 7pm, with call information shared via the Slack channel.
+
+## Our development process
+
+1. [Create a new GitHub branch.](https://github.com/codeforpdx/opentransit-metrics)
+2. Build your feature.
+3. Make a pull request. Request reviews from members of the team; you’ll need their approval to merge.
+
+- Good people to ask for reviews:
```

Comment:
We'll try to do a comprehensive update to this with new team members in another PR.